sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/StoreResponseBuilder.java

```diff
@@ -7,33 +7,31 @@
 import com.azure.cosmos.implementation.directconnectivity.WFConstants;
 
 import java.math.BigDecimal;
-import java.util.AbstractMap;
-import java.util.ArrayList;
-import java.util.List;
+import java.util.HashMap;
 import java.util.Map;
 
 import static com.azure.cosmos.implementation.Utils.getUTF8BytesOrNull;
 
 public class StoreResponseBuilder {
     private int status;
-    private List<Map.Entry<String, String>> headerEntries;
+    private Map<String, String> headers;
     private String content;
 
     public static StoreResponseBuilder create() {
         return new StoreResponseBuilder();
     }
 
     public StoreResponseBuilder() {
-        headerEntries = new ArrayList<>();
+        headers = new HashMap<>();
```

Comment:
Was there anything previously depending on header insertion ordering? If so, a `LinkedHashMap` may be a better option (though less performant)

Reply:
Nothing was depending on the ordering of the headers.